lib/mapreduce/index.js

```diff
@@ -680,6 +682,10 @@ function queryViewInQueue(view, opts) {
     var viewOpts = {
       descending : opts.descending
     };
+    opts.startkey = typeof opts.startkey  === 'undefined' ?
```

Comment:
Should we make this consistent with the other changes we've done similar to this? https://github.com/pouchdb/pouchdb/commit/f8629e3be86e619411598848b368ff9cd7846e56#diff-a7bcb5fcc46018435e1ce3fa65fcb992R653